src/files.rs

```diff
@@ -201,20 +106,21 @@ impl Files {
               (file_name)
             }
 
-            @if let Some(bytes) = bytes {
+            @if let Some(bytes) = entry.file_size {
               span class="filesize" {
                 (bytes.display_size())
               }
             }
-            @if file_type.is_file() && !config.paid() {
+            // if vfs.paid(file)
+            @if entry.file_type.is_file() && !self.vfs.paid(&dir.join_relative(entry.file_name.as_ref()).unwrap()) {
```

Comment:
This makes me think that maybe the entry should say whether it's paid or not:

```suggestion
            @if entry.file_type.is_file() && !entry.paid {
```